### PR TITLE
ADD VALIDATION IN RECURRING EXPENSE

### PR DESCRIPTION
**Pull request by ShivanshGairola**
Targets #2361 

### Description
1.Added a new request file
2.Add backend validation
### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by gautam071 — APPROVED** [review]
looks good

File: Modules/Expense/Http/Requests/recurringExpenseRequest.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        ];
+    }
+    public function messages()
+    {
+        return [
+            'name.required' => 'name is required',
+             'amount.required'=>'amount is required',
+             'currency.required'=>'currency is required',
+             'frequency.required'=>'frequency is required',
+        ];
```
Comment (GauravGusain98) [inline Modules/Expense/Http/Requests/recurringExpenseRequest.php:41]:
```suggestion
        return [
            'name.required' => 'name is required',
            'amount.required'=>'amount is required',
            'currency.required'=>'currency is required',
            'frequency.required'=>'frequency is required',
        ];
```

File: Modules/Expense/Http/Requests/recurringExpenseRequest.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Modules\Expense\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class recurringExpenseRequest extends FormRequest
```
Comment (GauravGusain98) [inline Modules/Expense/Http/Requests/recurringExpenseRequest.php:7]: The name of the Class is in incorrect syntax. 
It should be in the Title case `RecurringExpenseRequest`. Also, the file name should also in title case. You can take reference from other request files

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**